### PR TITLE
猪脚饭

### PR DESCRIPTION
**Pull request by immoonancient**
#277

### CODE REVIEW

File: subtitles/dao-yue-she/20191007-zhu-jiao-fan.sbv [hunk 1]
```diff
@@ -0,0 +1,951 @@
...
+# 你们还没吃呢
+You haven't had lunch yet
+
+0:06:12.340,0:06:12.900
+# 饿
+Yes
+
+0:06:12.900,0:06:14.200
+# 你们现在应该点什么？
+So, what are you supposed to point at?
```
Comment (NoraZheng) [inline subtitles/dao-yue-she/20191007-zhu-jiao-fan.sbv]: So, what are you supposed to order /click now?

File: subtitles/dao-yue-she/20191007-zhu-jiao-fan.sbv [hunk 2]
```diff
@@ -0,0 +1,951 @@
...
+# 点关注
+The subscribe button
+
+0:06:16.020,0:06:17.120
+# 哦～～
+Oh~~
+
+0:06:17.120,0:06:18.960
+# 你以为是让他们点外卖啊
+You thought I'm asking them to order food?
```
Comment (NoraZheng) [inline subtitles/dao-yue-she/20191007-zhu-jiao-fan.sbv]: You thought I was asking them to order food?

File: subtitles/dao-yue-she/20191007-zhu-jiao-fan.sbv [hunk 3]
```diff
@@ -0,0 +1,951 @@
...
+# 可能是茶叶和我的茶具
+might be tea and accessories
+
+0:06:43.300,0:06:44.640
+# 是不是三分样
+Regardless of your actual ability
+
+0:06:44.640,0:06:46.220
+# 先把自己的一些装备先凑齐
+you got to prepare all the equipments first
```
Comment (NoraZheng) [inline subtitles/dao-yue-she/20191007-zhu-jiao-fan.sbv]: equipment 不可数

File: subtitles/dao-yue-she/20191007-zhu-jiao-fan.sbv [hunk 4]
```diff
@@ -0,0 +1,951 @@
...
+
+0:06:46.220,0:06:47.160
+# 就像健身一样
+It's like going to gym
+
+0:06:47.280,0:06:48.600
+# 先把装备买好了
+You have to buy all the equipments first
```
Comment (NoraZheng) [inline subtitles/dao-yue-she/20191007-zhu-jiao-fan.sbv]: equipement

Comment (NoraZheng) [reply subtitles/dao-yue-she/20191007-zhu-jiao-fan.sbv]: gear 装备

File: subtitles/dao-yue-she/20191007-zhu-jiao-fan.sbv [hunk 5]
```diff
@@ -0,0 +1,951 @@
+# https://www.youtube.com/watch?v=95Ydh6HXUQQ
+
+# 大叔掏出猪肘子，剁碎浇汁给我吃，一盘蒜2碗饭香糯下饭！| 潮汕美食 | 猪脚饭
+# This uncle serves Chaoshan pig feet meal. Chopped and mixed with juice and garlic, super tasty!
```
[the remark this answers is not in the document]
Comment (immoonancient) [reply subtitles/dao-yue-she/20191007-zhu-jiao-fan.sbv]: 那就不改了...

**Pull request merged**